website/src/components/Edit/EditPage.tsx

```diff
@@ -29,13 +30,16 @@ const logger = getClientLogger('EditPage');
 const InnerEditPage: FC<EditPageProps> = ({
     organism,
     dataToEdit,
+    segmentNames,
     clientConfig,
     accessToken,
     groupedInputFields,
     submissionDataTypes,
 }) => {
     const [editableMetadata, setEditableMetadata] = useState(EditableMetadata.fromInitialData(dataToEdit));
-    const [editableSequences, setEditableSequences] = useState(EditableSequences.fromInitialData(dataToEdit));
+    const [editableSequences, setEditableSequences] = useState(
+        EditableSequences.fromInitialData(dataToEdit, segmentNames),
```

Comment:
I think the issue is here - we do this for all segments and previously it was only for all segments that were also in the input file